gcs/pom.xml

```diff
@@ -164,6 +164,11 @@
       <groupId>org.assertj</groupId>
       <artifactId>assertj-core</artifactId>
     </dependency>
+    <dependency>
```

Comment:
We use Google Flogger for logging, please remove this dependency.

Reply:
Updated